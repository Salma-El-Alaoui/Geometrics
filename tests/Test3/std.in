L l1 1 1 1 1
L l2 2 2 2 2
L l3 3 3 3 3
L l2 1 2 3 4
L l4 a 1 1 1
L l4 -1 -1 -1 -1
L l5 1 1
L l5 1 1 1 1 1 1
L 123 1 2 3 4
EXIT
