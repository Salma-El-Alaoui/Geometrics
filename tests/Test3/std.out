OK
#Element ajoute: l1
OK
#Element ajoute: l2
OK
#Element ajoute: l3
ERR
#Un element geometrique nomme l2 existe deja
ERR
#Parametres invalides
OK
#Element ajoute: l4
ERR
#Parametres invalides
ERR
#Parametres invalides
ERR
#Parametres invalides
